### DISCUSSION TITLE
Data Channel authentication technologies

### DISCUSSION THREAD

**Discussion by castroflavio** [scistream/scistream-proto#36] [Ideas]
Are there alternatives to Stunnel?

Should we explore L3 tunnels such as VPNs and Wireguard?

We don't know if they would perform better.

**Comment by JoaquinChung** [comment]
Would this be L3 tunnels in software?

**Comment by castroflavio** [comment]
Yes. But I'm pretty sure, L3 tunnels can also be implemented in hardware at wire-speeds.
